packages/shared/graphql/clientIdHeader.ts

```diff
@@ -0,0 +1,14 @@
+import { v4 as uuid } from "uuid";
+
+// Generate a client ID so that we can easily filter backend requests down to a specific client
+// browser tab. It might seem like we should just be attaching the session ID here, but devtools
+// makes requests before the session is established, so we use an arbitrary ID instead.
+export const GRAPHQL_CLIENT_ID = uuid();
+// The header value of this header key will be logged on the backend during GraphQL requests.
+export const GRAPHQL_CLIENT_ID_HEADER_KEY = "Replay-Client-Id";
```

Comment:
Why are we exporting these keys like this if they're only being reference in one place (below)

Reply:
I could use changes in this PR and taking over. Talked to Josh (here at offsite) and removed the unused exports.

Reply:
Okedoke.